jfxtras-agenda/src/main/java/jfxtras/internal/scene/control/skin/agenda/base24hour/AgendaSkinTimeScale24HourAbstract.java

```diff
@@ -238,7 +241,15 @@ public void refresh() {
 		refreshLocale();
 		setupAppointments();
 		nowUpdateRunnable.run(); 
-	}
+	}
+
+	/**
+	 * 
+	 */
+	@Override
+    public Pane getNodeForPopup(Appointment appointment) { return appointmentNodeMap.get(appointment); }
+    final private Map<Appointment, Pane> appointmentNodeMap = new WeakHashMap<>();
```

Comment:
I suppose it would be in a different class, AgendaSkinTimeScale24HourAbstract?  I'll have to think about it later.  It's too late for me and I need some sleep.  We are almost half-way around the world from each other.  You are just beginning your day.  I'm at the end of mine.